### PR TITLE
Ker 209 hds 3 header

### PR DESCRIPTION
**Pull request by timwessman**
This PR replaces the old Header component with HDS 3.0.0 Header component.
[KER-209](https://helsinkisolutionoffice.atlassian.net/browse/KER-209)

How to test:
- Check that the new Header looks OK. It should have a generic HDS style
- Try change languages
- Try to login/logout
- Try mobile screen size and open the navigation menu

[KER-209]: https://helsinkisolutionoffice.atlassian.net/browse/KER-209?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/components/Header/Header.jsx [hunk 1]
```diff
@@ -0,0 +1,147 @@
+/* eslint-disable react/no-multi-comp */
```
Comment (mikkojamG) [inline src/components/Header/Header.jsx:1]: This ignore must have been necessary at some point, but looks like it's not needed anymore 🤔

File: src/components/Header/Header.jsx [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+
+// eslint-disable-next-line import/no-unresolved
+import logoSwedishBlack from '@city-images/logo-sv-black.svg';
+
+import userManager from "../../utils/userManager";
+import { getUser } from '../../selectors/user';
+import config from "../../config";
+import { localizedNotifyError } from '../../utils/notify';
+
+class Header extends React.Component {
```
[the remark this answers is not in the document]
Comment (timwessman) [reply src/components/Header/Header.jsx]: Sure. Fixed

File: src/components/Header/Header.jsx [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+    } catch (error) {
+      localizedNotifyError("loginAttemptFailed");
+    }
+  }
+
+  onLanguageChange = (newLanguage) => {
+    if (newLanguage !== this.props.language) {
+      const languageParam = `lang=${newLanguage}`;
+      let searchParams;
+      if (window.location.search.includes('lang=')) {
```
[the remark this answers is not in the document]
Comment (timwessman) [reply src/components/Header/Header.jsx]: Fixed this to use `URL API`'s `URLSearchParams`

**Comment by timwessman** [comment]
Fixed all small changes.
Removed some unused css. Turku probably used some of these old navigation's css classes to customize their header, but as the component is totally changed, they will need to fix their customization also.

**Review by mikkojamG — APPROVED** [review]
LGTM 👍

**Pull request merged**